sponsorship_compassion/models/contracts.py

```diff
@@ -66,8 +66,8 @@ class SponsorshipContract(models.Model):
     transfer_partner_id = fields.Many2one(
         "compassion.global.partner", "Transferred to", readonly=False
     )
-    global_id = fields.Char(
-        help="Connect global ID", readonly=True, copy=False, tracking=True
+    gmc_commitment_id = fields.Char(
+        help="Connect global ID", readonly=True, copy=False, tracking=True, oldname="global_id"
```

Comment:
`oldname` is no longer supported in v13. You should replace by a script (using openupgrade library) in pre-migration script.
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

Reply:
In my local tests that did work but i'll look in the new method to do that.